tests/foreman/smoke/test_ui_smoke.py

```diff
@@ -199,8 +199,10 @@ def test_positive_smoke(self):
                     common_locators['alert.success']
                 ))
             # Add puppet-module to content-view
-            self.content_views.add_puppet_module(
-                cv_name, 'httpd', filter_term='Latest')
+            if not bz_bug_is_open(1297308):
```

Comment:
Yes, We are skipping a portion of smoke-test which adds the puppet-module to CV. Ideally, smoke tests are meant for build health check. So if a test or subset of test is failing then we should leave as is. That means build is not good.

On other hand, since our automation-trigger, triggers the automation jobs only if the smoke tests are green then we may skip the failed sub-test with bz decorator instead of entire test.